blackboard_sync/universities.json

```diff
@@ -670,6 +670,16 @@
     },
     "api_url": "https://mstc.blackboard.com"
   },
+  {
+    "name": "École supérieure privée d'ingénierie et de technologie",
```

Comment:
There is only a small problem: your app is not compatible with special French characters.
So, I suggest renaming it to: `Ecole superieure privee d'ingenierie et de technologie`

Reply:
What do you mean it is not compatible? Is there an error with the UI?